cypress/integration/check_summary/enabled-clusters.spec.js

```diff
@@ -21,19 +21,22 @@ describe('Testing enabled clusters amount', () => {
         const num1 = parseFloat($div.text())
         cy.fixture('data').then((data) => {
           cy.get('.q-page-container > div')
-          .children()
-          .should('contain', data.cluster1)
+            .children()
+            .should('contain', data.cluster1)
         })
-        cy.get('div').contains('General').click({force: true})
-        cy.get('div').children().contains('Server').its('length').then(res=>{
-          if(res > 0){
-            cy.get('div').children().contains('Not Enabled').first().click()
-            cy.get('.q-virtual-scroll__content > :nth-child(3)')
-              .contains('Server')
-              .click()
-          }
-      });
-        
+        cy.get('div').contains('General').click({ force: true })
+        cy.get('div')
+          .children()
+          .contains('Server')
+          .its('length')
+          .then((res) => {
+            if (res > 0) {
+              cy.get('div').children().contains('Not Enabled').first().click()
+              cy.get('.q-virtual-scroll__content > :nth-child(3)')
+                .contains('Server')
+                .click()
+            }
+          })
       })
```

Comment:
Seems like the auto formatter changed content here. Get rid of all auto formatting so that we are looking at just the actual changes related to this issue. There are other files in this PR which have auto formatter as well. Clean those up too.